### DISCUSSION TITLE
Backend Shop Suggestions

### DISCUSSION THREAD

**Discussion by elusive-ttv** [gameserverapp/Platform#51] [Suggestions]
I'd like to see some organization in the backend of the shop, like separated by cluster availability (obviously things available in both would show up on both lists) and maybe sections to organize different items like weapons, armor, creatures, etc.

**Comment by mvaessen** [comment]
Hi, we have a number of things in place that I think might be what you're looking for.

You can use a **Custom label** on the shop pack settings page, as well as the "Where can this be delivered?" section to determine where people can order it. Both can be found on the Shop pack settings section.

**Comment by elusive-ttv** [comment]
Yes, that is very helpful. But on the backend creating shop packs, it would be amazing if we could filter by cluster! Right now I have hundreds of packs for 2 different clusters that I can't filter to be able to see what each specific cluster has.

**Comment by mvaessen** [comment]
Hi, have you considered using the Custom label?

You can use the "Where can this be delivered" to have it show up only for the particular cluster. This will also activate the filter on the community website, allowing people to filter per cluster.

**Comment by mvaessen** [comment]
Addressing in #612
